### PR TITLE
Add jian/keymaps/left_hand

### PR DESCRIPTION
**Pull request by KGOH**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by KGOH** [comment]
Travis tries to compile every keymap and fails with 
"The job exceeded the maximum time limit for jobs, and has been terminated." 
I'm not sure if this is problem with travis or with my PR

**Comment by drashna** [comment]
Nope, that's not a problem with your PR. As long as your keyboard compiles, that's the important part.

File: keyboards/jian/keymaps/left_hand/keymap.c [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/jian/keymaps/left_hand/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/jian/rev2/readme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This is GB revision
```
Comment (noroadsleft) [inline keyboards/jian/rev2/readme.md:1]: I feel like `keyboards/jian/readme.md` should call out that the rev2 is the GB revision.

**Pull request merged**